### ISSUE TITLE
Grunt Compile Error

### ISSUE THREAD

**Issue by admazzola** [michaelgrosner/tribeca#3]
I got this error after following all of the steps and then running 'grunt compile':

~/tribeca$ sudo grunt compile
module.js:338
    throw err;
          ^
Error: Cannot find module './lexer'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/tribeca/node_modules/grunt/node_modules/coffee-script/lib/coffee-script/coffee-script.js:10:10)
    at Object.<anonymous> (/tribeca/node_modules/grunt/node_modules/coffee-script/lib/coffee-script/coffee-script.js:167:4)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)

I know my node is the correct version (0.12).  The one weird thing is that I installed it under 'sudo' but it should still work... what is this 'lexer' module and why cant it be found?

**Comment by michaelgrosner** [comment]
Yikes, I have no idea what that could be coming from. It's not in the tribeca `package.json`. It's most likely a dependency of one of my dependencies. One link I found on Google is: http://stackoverflow.com/q/24639586/178073

I also updated the repository with a docker-based compile+deploy strategy. If you're amenable to trying that out, it hopefully should reduce the chances of these weird compile bugs.
